Source/WebKit/UIProcess/RemoteLayerTree/RemoteScrollingCoordinatorProxy.cpp

```diff
@@ -245,6 +245,11 @@ bool RemoteScrollingCoordinatorProxy::scrollingTreeNodeRequestsScroll(ScrollingN
     return false;
 }
 
+bool RemoteScrollingCoordinatorProxy::scrollingTreeNodeRequestsKeyboardScroll(ScrollingNodeID scrolledNodeID, const RequestedKeyboardScrollData&)
+{
+    return scrolledNodeID == rootScrollingNodeID();
```

Comment:
Does this make sense on macOS? The function above is the way it is because the root is special on iOS (because it's the main UIScrollView).